app/initializers/authenticators.js

```diff
@@ -98,30 +98,47 @@ export default {
 			env = purecloudEnvironmentTld();
 		}
 
-		if (!env || env === "null") { // type of variable "env" is string, so needs to check "null" instead of null
+		if (!env || env === 'null') {
+			// type of variable "env" is string, so needs to check "null" instead of null
 			var that = this;
-			document.getElementById("regionModal").style.display = "block";
-			$('.regionButton').on("click", function () {
-				env = $(this).attr("value");
+
+			// Append lower envs
+			const lowerEnvRegex = /inin[dt]ca|localhost/i;
```

Comment:
This block is the only change; everything else is formatting from enforcing prettier rules.